src/writing-policies/go.md

```diff
@@ -1,84 +0,0 @@
-> **Note well:** WebAssembly and WASI are fast evolving targets. The contents
```

Comment:
Do not remove it, just leave it out of the `SUMMARY.md` file. I'll fix it with another PR